### ISSUE TITLE
Suppress warnings if disabled

### ISSUE THREAD

**Issue by amk221** [TrackJs/trackjs-package#18]
I have TrackJs disabled when our test suite is run, but TrackJs still attempts to look for a token and warns if one isn't found. Outputting in our logs (which isn't a problem, just a bit annoying)

```
  ---
        Log: |
            { type: 'warn', text: '\'TrackJS could not find a token\'\n' }
    ...
```

**Comment by BrandesEric** [comment]
Ah yes, I can see where that could be annoying :)  If you'd like a quick workaround, you can simply put in a garbage value for the token:

```js
_trackJs = {
  token: 'a',
  enabled: false
}
```

I know that's sort of hacky, but I'm not sure when we'll be able to get around to fixing it.  I have filed a bug on our internal tracker though :)

**Comment by toddhgardner** [comment]
Hi @amk221, this has been resolved in Tracker 2.7.0
